packages/@lwc/engine/src/framework/api.ts

```diff
@@ -99,9 +99,18 @@ const CHAR_G = 103;
 const NamespaceAttributeForSVG = 'http://www.w3.org/2000/svg';
 const SymbolIterator = Symbol.iterator;
 
+function validateVNodeAfterElmCreation(vnode: VNode) {
```

Comment:
this is a generalization, we could eventually move more things into this validation mechanism. It is abstracted for any kind of node.